File: InCoqnito.Kalender.Models/Models/Request/InvitationRequest.cs
using InCoqnito.Kalender.Models.DTOs;
using System;
using System.Collections.Generic;

namespace InCoqnito.Kalender.Models.Models.Request
{
    public class InvitationRequest
    {
        public InvitationRequest()
        {
            SharedEmails = new List<EmployeeDto>();
        }
        public int AuthorId { get; set; }
        public string Date { get; set; }
        public string StartTime { get; set; }
        public string EndTime { get; set; }
        public string Description { get; set; }
        public List<EmployeeDto> SharedEmails { get; set; }
    }
}

File: InCoqnito.Kalender.Web/ClientApp/kalenderApp/src/app/models/InvitationRequest.ts
import { IEmployee } from './IEmployeeDto';

export class InvitationRequest{
    public authorId: number;
    public date: string;
    public startTime: string;
    public endTime: string;
    public description: string;
    public sharedEmails: IEmployee[];
    constructor(authorId = 0, date = '', startTime = '', endTime = '', description = '', sharedEmails = []) {
      this.authorId = authorId;
      this.date = date;
      this.startTime = startTime;
      this.endTime = endTime;
      this.description = description;
      this.sharedEmails = sharedEmails;
    }
}
